src/robot_manager.cpp

```diff
@@ -557,6 +574,61 @@ bool RobotManager::setTargetCb(temoto_robot_manager::RobotSetTarget::Request& re
   return true;
 }
 
+
+
+bool RobotManager::getTargetCb(temoto_robot_manager::RobotGetTarget::Request& req,
+                               temoto_robot_manager::RobotGetTarget::Response& res)
+{
+  if (active_robot_->isLocal())
+  {
+    TEMOTO_INFO("Request: '%s'", req.ref_joint.c_str());
+    TEMOTO_INFO("Request: '%s'", req.respect_to.c_str());
+    res.pose = active_robot_->getTarget();
+  }
+  else
+  {
+    TEMOTO_INFO("robot is not local");
+  }
+
+  return true;
+}
+
+
+bool RobotManager::goalCb(temoto_robot_manager::RobotGoal::Request& req, temoto_robot_manager::RobotGoal::Response& res)
+{
+  active_robot_->goal("map", req.target_pose);
+  std::string act_rob_ns = active_robot_->getConfig()->getAbsRobotNamespace() + "/move_base";
+
+  //MoveBaseClient ac("/xarm7_temoto/robot_manager/robots/clearbot/move_base", true);   //For testing with clearbot
+  MoveBaseClient ac(act_rob_ns, true);  
+  
+  
+  while(!ac.waitForServer(ros::Duration(5.0))){
+    ROS_INFO("Waiting for the move_base action server to come up");
+  }
+  
+  move_base_msgs::MoveBaseGoal goal;  
+  goal.target_pose.pose = req.target_pose.pose;
+  TEMOTO_INFO_STREAM(goal.target_pose); 
+  goal.target_pose.header.frame_id = "map";
+  goal.target_pose.header.stamp = ros::Time::now();
+ 
+ 
+  TEMOTO_INFO_STREAM(goal.target_pose); 
+
+  ac.sendGoal(goal);
+  ac.waitForResult();
+
+  if(ac.getState() == actionlib::SimpleClientGoalState::SUCCEEDED)
```

Comment:
Even if you have just 1 line in your `if/else` statement, surround them by `{ }`. It does not change the functionality but this is the coding convention in this repo.

Reply:
Agree, good programming practices.